test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -284,7 +284,7 @@ protected IEnumerable<NodeModel> AllNodes
 
         private void CurrentUIApplication_ViewActivating(object sender, ViewActivatingEventArgs e)
         {
-            ((RevitDynamoModel)this.ViewModel.Model).SetRunEnabledBasedOnContext(e.NewActiveView);
+            ((RevitDynamoModel)this.ViewModel.Model).SetRunEnabledBasedOnContext(e.NewActiveView, true);
```

Comment:
depending on how you handle the change of this method can we also make sure the test the other options - either the default call or the old obsoleted method?